src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -296,6 +362,216 @@ const formConfig = {
             }
           }
         },
+        // TODO: Should this be privateMedicalRecordsRelease or privateRecordsRelease? 
+        privateMedicalRecordsRelease: {
+          title: '',
+          path: 'supporting-evidence/:index/private-medical-records-release',
+          showPagePerItem: true,
+          arrayPath: 'disabilities',
+          depends: (formData, index) => {
+            const hasRecords = _.get(`disabilities.${index}.view:privateMedicalRecords`, formData);
+            // TODO: enable once previous chapter merged
+            // const requestsRecords = !_.get(`disabilities.${index}.view:uploadPrivateRecords`, formData);
+            const requestsRecords = true;
+            return hasRecords && requestsRecords;
+          },
+          uiSchema: {
+            disabilities: {
+              items: {
+                'ui:description': 'Please let us know where and when you received treatment. We’ll request your private medical records for you. If you have your private medical records available, you can upload them later in the application',
+                treatments: {
+                  'ui:options': {
+                    itemName: 'Record',
+                    viewField: () => {
+                      return (<div>hello</div>);
+                    }
+                  },
+                  items: {
+                    treatment: {
+                      'ui:order': ['treatmentCenterName', 'privateMedicalRecordsReleaseAccepted', 'view:privateMedicalRecordsReleasePermissionRestricted', 'startTreatment', 'endTreatment', 'treatmentCenterCountry', 'treatmentCenterStreet1', 'treatmentCenterStreet2', 'treatmentCenterCity', 'treatmentCenterState', 'treatmentCenterPostalCode', 'treatmentCenterPhone'],
+                      treatmentCenterName: { // TODO: is this required?
+                        'ui:title': 'Name of private provider or hospital'
+                      },
+                      privateMedicalRecordsReleaseAccepted: {
+                        'ui:title': 'I give my consent, or permission, to my doctor to only release records related to this condition'
+                      },
+                      'view:privateMedicalRecordsReleasePermissionRestricted': {
+                        'ui:description': () => {
+                          return (<div className="usa-alert usa-alert-warning no-background-image">
+                            <span>Limiting consent means that your doctor can only share records that are directly related to your condition. This could add to the time it takes to get your private medical records.</span>
+                          </div>);
+                        },
+                        'ui:options': {
+                          expandUnder: 'privateMedicalRecordsReleaseAccepted' // TODO: prevent auto
```

Comment:
@jbalboni This has the unfortunate habit of expanding automagically. I can't remember...did we have to do some sort of tomfoolery to prevent this, or is this perhaps because we're in an array page (seems odd, but I never know)?

Reply:
This requires 'ui:collapsed': true, which yields the expected behavior.

Reply:
Excellent! Looks like that isn't in the documentation and either should be, or when `expandUnder` has a value, `ui:collapsed` should default to `true` (at which point, it should be documented that to prevent this behavior, we'd use `'ui:collapsed': false`).